projection/src/main/scala/akka/projection/r2dbc/internal/R2dbcOffsetStore.scala

```diff
@@ -796,18 +817,20 @@ private[projection] class R2dbcOffsetStore(
         val until = currentState.latestTimestamp.minus(settings.timeWindow)
         val minSlice = timestampOffsetBySlicesSourceProvider.minSlice
         val maxSlice = timestampOffsetBySlicesSourceProvider.maxSlice
+        val notInLatestBySlice = currentState.latestBySlice.map { record =>
+          s"${record.slice}-${record.pid}-${record.seqNr}"
+        }.toArray
```

Comment:
So up to 1024 slice entries depending on what the projection saw?

Reply:
yes, up to the number of slices in the slice range that the offset store is responsible for (there is one offset store instance per projection instance)